server/routes/user.js

```diff
@@ -31,6 +31,10 @@ router.route('/:userId/device/:deviceId')
 
   .delete(validate(paramValidation.deviceById), userCtrl.removeDevice);
 
+router.route('/:userId/group')
```

Comment:
I'm going with group because our convention is using singular. for example `GET api/v2/group` would give a list of all groups.